Magento_Theme/templates/myknockoutcomponent.phtml

```diff
@@ -0,0 +1,17 @@
+<div data-bind="scope: 'my.scope'">
+    <!-- ko template: 'Magento_Theme/mytemplate' --><!-- /ko -->
+</div>
+
+<script type="text/x-magento-init">
+    {
+        "*": {
+            "Magento_Ui/js/core/app": {
+                "components": {
+                    "my.scope": {
```

Comment:
don't use dots inside the component name. you could break the logic of chaining logic for components naming with nesting parent.childLvl1.childLvl2